third_party/intel/backend/driver.py

```diff
@@ -14,36 +14,35 @@
 from packaging.specifiers import SpecifierSet
 
 
-def find_sycl(include_dir: list[str]) -> tuple[list[str], list[str]]:
+def find_sycl(include_dir: list[str]) -> tuple[list[str], str]:
```

Comment:
Since libsycl location can be None:
```suggestion
def find_sycl(include_dir: list[str]) -> tuple[list[str], Optional[str]]:
```